source/transactions/transactions.rst

```diff
@@ -290,6 +302,17 @@ See `Why is readPreference part of TransactionOptions?`_.
 In the future, we might relax this restriction and allow any read
 preference on a transaction.
 
+maxCommitTimeMS
+^^^^^^^^^^^^^^^
+
+The maximum amount of time to allow a single commitTransaction command to run.
+
+This option is only sent with the commitTransaction command(s) and only if the
+caller explicitly provides a value. The default is to not send a value.
+
+Note, this option is an alias for the `maxTimeMS` commitTransaction command
```

Comment:
Double backticks for code formatting on `maxTimeMS` here.

Reply:
Done.